base/bitarray.jl

```diff
@@ -1291,11 +1291,6 @@ for f in (:div, :mod)
     end
 end
 
-function (&)(B::BitArray, x::Bool)
```

Comment:
This will lead to a performance regression, since the corresponding `.&` version will have to check each element?